content/introduction-and-setup/walkthrough/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Walkthrough"
```

Comment:
What is the purpose of this page?  Seems Redundant.  Could be added to Setup for C#.

Reply:
Every folder has to have an index.md file which is why we have to keep it.

Reply:
Let me know if there is anything I can do to help it stand out from the crowd!

Reply:
Can you please rename Walkthrough to Setup for C# and delete the subfolder Setup for C#, they seem to have the same information.

Reply:
Done! Didn't want to delete it unless you approved since Git and deletion can turn into such a mess.

Reply:
Thanks Sally, Looks good!

Reply:
Thanks!